lnwallet/channel_test.go

```diff
@@ -5443,6 +5443,158 @@ func TestMaxAcceptedHTLCs(t *testing.T) {
 	}
 }
 
+// TestMaxAsynchronousHtlcs tests that Bob correctly receives (and does not
+// fail) an HTLC from Alice when exchanging asynchronous payments. We want to
+// mimic the following case where Bob's commitment transaction is full before
+// starting:
+// 	Alice                    Bob
+// 1.         <---settle/fail---
+// 2.         <-------sig-------
+// 3.	       --------sig------> (covers an add sent before step 1)
+// 4. 	       <-------rev-------
```

Comment:
nit: this line and line above are indented improperly

Reply:
squashed